Formula/java-service-wrapper.rb

```diff
@@ -0,0 +1,19 @@
+class JavaServiceWrapper < Formula
+  desc "Simplify the deployment, launch and monitoring of Java applications"
+  homepage "https://wrapper.tanukisoftware.com/"
+  url "https://downloads.sourceforge.net/project/wrapper/wrapper_src/Wrapper_3.5.46_20210903/wrapper_3.5.46_src.tar.gz"
+  sha256 "82e1d0c85488d1389d02e3abe3359a7f759119e356e3e3abd6c6d67615ae5ad8"
+  license "GPL-2.0-only"
+
+  depends_on "openjdk@11" => :build
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@11"].opt_prefix
+    system "ant", "-Dbits=64", "-Djavac.target.version=1.6"
```

Comment:
A comment explaining the `1.6` here would be good.

Reply:
Ok, done.

Reply:
Is there a potential risk something breaks as upstream shipped with version 1.4 and probably never tested the build with 1.6? I am a little bit worried of hard-to-debug compile / runtime exceptions.

Reply:
In my opinion, the risk is limited. It all depends on your context, if this is for a development environment the risk is acceptable, if this is for a mission critical application, you should buy a [server license](https://wrapper.tanukisoftware.com/doc/english/licenseServer.html) and use the official build.

Maybe we can add a sentence in the caveat about this ?

Reply:
There's no need; that caveat would apply to everything we ship.